float32 speed1
float32 speed2
bool driverstate
